R/add_phylopic_base.r

```diff
@@ -45,15 +48,27 @@
 #'   add_phylopic_base(cat, posx[i], posy[i], size[i], alpha=.8)
 #' }
 #' }
-add_phylopic_base <- function(img, x = NULL, y = NULL, ysize = NULL, 
+add_phylopic_base <- function(img, x = NULL, y = NULL, ysize = NULL, xsize=NULL,
                               alpha = 0.2, color = NULL) {
   # color and alpha the animal
   img <- recolor_phylopic(img, alpha, color)
 
   # work out the dimensions of the image
   dims <- dim(img)[1:2]
-  AR <- dims[1] / dims[2]
-  xsize <- AR * ysize
+
+
+  # compute the size
+  if(missing(xsize)){
```

Comment:
@dill  AFAIK `missing()` would be used if the parameter in the function definition did not have a default value, wheres xsize does in this case. I'd think `if (is.null(xsize))` would be more appropriate

Reply:
oh right, yeah, just on autopilot. Fixed in 1b3bfb8